client-src/src/pages/counselor/inq-details/inq-details.ts

```diff
@@ -179,7 +179,22 @@ export class InqDetailsPage {
         );
     }else{
       this.notify.showError(this.message.FORM.INVALID);
+      console.log(this.inqForm);
       this.helper.markInvalidFields(this.inqForm);
+      this.helper.markInvalidSelect(this.inqForm, 'gender');
+      this.helper.markInvalidSelect(this.inqForm, 'dob');
+      this.helper.markInvalidSelect(this.inqForm, 'hQualification');
+      this.helper.markInvalidSelect(this.inqForm, 'computerKnowledge');
+      this.helper.markInvalidSelect(this.inqForm, 'areaOfInterest');
+      if(this.areas && this.inqForm.value.address.pin.length == 6)this.helper.markInvalidSelect(this.inqForm.get('address'), 'locationId');
+      this.helper.markInvalidSelect(this.inqForm.get('education').at(0), 'educationQualification');
```

Comment:
This may throw null pointer exception if .get('education') itself is null. Should handle that and similar scenario

Reply:
We have defined it in the constructor, it cannot be null.